### PR TITLE
Add workspace to disk-virt-[customize|sysprep]

### PR DESCRIPTION
**Pull request by 0xFelix**
**What this PR does / why we need it**:

This allows to pass customization data or secrets to be consumed during
the run of virt-customize or virt-sysprep.

**Special notes for your reviewer**:

This will be especially useful when https://github.com/tektoncd/pipeline/pull/5085 is merged.

**Release note**:

```release-note
Add workspace to disk-virt-customize and disk-virt-sysprep
```

### CODE REVIEW

File: tasks/disk-virt-customize/examples/taskruns/disk-virt-customize-taskrun-workspace.yaml [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+    - name: customizeCommands
+      value: |
+        update
+        install make,ansible
+        delete /var/cache/dnf
+        root-password file:/data/password
+  workspaces:
+    - name: data
+      secret:
+        name: disk-virt-customize-taskrun-workspace
```
Comment (ksimon1) [inline tasks/disk-virt-customize/examples/taskruns/disk-virt-customize-taskrun-workspace.yaml:29]: please add new line at the end of file

File: templates/disk-virt-sysprep/examples/disk-virt-sysprep-taskrun-workspace.yaml [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+    - name: sysprepCommands
+      value: |
+        update
+        install make,ansible
+        operations firewall-rules,defaults
+        root-password file:/data/password
+  workspaces:
+    - name: data
+      secret:
+        name: {{ item }}
```
Comment (ksimon1) [inline templates/disk-virt-sysprep/examples/disk-virt-sysprep-taskrun-workspace.yaml:32]: same here

Comment (0xFelix) [reply templates/disk-virt-sysprep/examples/disk-virt-sysprep-taskrun-workspace.yaml]: Done

File: tasks/disk-virt-customize/examples/taskruns/disk-virt-customize-taskrun-workspace.yaml (hunk 1 above)
Comment (0xFelix) [reply tasks/disk-virt-customize/examples/taskruns/disk-virt-customize-taskrun-workspace.yaml]: Done

**Pull request merged**